src/vanilla.ts

```diff
@@ -13,6 +13,12 @@ export const getDefaultStore: GetDefaultStore = import.meta.env?.USE_STORE2
   ? store2.getDefaultStore
   : store.getDefaultStore
 
+export type {
+  INTERNAL_DevStoreRev2,
+  INTERNAL_DevStoreRev3,
+  INTERNAL_PrdStore,
+} from './vanilla/store.ts'
+
```

Comment:
We want to avoid exporting internal types from the entry point.
(As of now), we can directly import those types from `'jotai/vanilla/store'`

```suggestion
```